README.md

```diff
@@ -1 +1,47 @@
-# django-audittrail - Register and store users actions in database
+# Django-AuditTrail - Register and store users actions in database
+
+This project allow you register and store users actions in database and then do whatever you want with this data.
```

Comment:
It would be better to say: "allows tracking user actions within a Django powered system."